src/open_inwoner/scss/components/Header/AnchorMenu.scss

```diff
@@ -63,22 +63,47 @@
     margin-left: var(--spacing-large);
   }
 
-  .link {
+  &__list-item {
     box-sizing: border-box;
-    padding: var(--spacing-large) var(--spacing-medium);
+    font-weight: normal;
+    margin: 0;
 
     @media (min-width: 768px) {
-      border-left: var(--border-width) solid;
-      border-color: var(--color-gray-light);
-      padding: var(--spacing-large);
+      margin: 0;
     }
   }
 
-  &__list-item--active .link {
+  &__list-item .link {
+    padding: 0.75rem 0 0.75rem 1.5rem;
```

Comment:
Also: no need for magic numbers here. Use the spacing that is already used in the anchor-menu; so something a little more like this: "var(--spacing-medium) 0 var(--spacing-medium) var(--spacing-medium)"